src/commands/deployWorkspaceProject/deploymentConfiguration/workspace/azureResources/AzureResourceVerifyStepBase.ts

```diff
@@ -5,22 +5,22 @@
 
 import { GenericTreeItem, activityFailContext, activityFailIcon, activitySuccessContext, activitySuccessIcon } from "@microsoft/vscode-azext-utils";
 import { type Progress } from "vscode";
-import { ExecuteActivityOutputStepBase, type ExecuteActivityOutput } from "../../../../utils/activity/ExecuteActivityOutputStepBase";
-import { createActivityChildContext } from "../../../../utils/activity/activityUtils";
-import { localize } from "../../../../utils/localize";
-import { type DeploymentConfigurationSettings } from "../../settings/DeployWorkspaceProjectSettingsV2";
-import { type WorkspaceDeploymentConfigurationContext } from "./WorkspaceDeploymentConfigurationContext";
+import { ExecuteActivityOutputStepBase, type ExecuteActivityOutput } from "../../../../../utils/activity/ExecuteActivityOutputStepBase";
+import { createActivityChildContext } from "../../../../../utils/activity/activityUtils";
+import { localize } from "../../../../../utils/localize";
+import { type DeploymentConfigurationSettings } from "../../../settings/DeployWorkspaceProjectSettingsV2";
+import { type WorkspaceDeploymentConfigurationContext } from "../WorkspaceDeploymentConfigurationContext";
 
 export abstract class AzureResourceVerifyStepBase extends ExecuteActivityOutputStepBase<WorkspaceDeploymentConfigurationContext> {
     public abstract priority: number;
 
     protected abstract resourceType: 'resource group' | 'container app' | 'container registry';
-    protected abstract deploymentSettingsKey: string;
-    protected abstract contextKey: string;
+    protected abstract deploymentSettingsKey: keyof DeploymentConfigurationSettings;
+    protected abstract contextKey: keyof WorkspaceDeploymentConfigurationContext;
 
     protected async executeCore(context: WorkspaceDeploymentConfigurationContext, progress: Progress<{ message?: string | undefined; increment?: number | undefined }>): Promise<void> {
         this.options.shouldSwallowError = true;
-        progress.report({ message: localize(`verifyingResourceType`, 'Verifying {0}...', this.resourceType) });
+        progress.report({ message: localize(`verifyingAzureResources`, 'Verifying Azure resources...') });
```

Comment:
I thought it was nice to include the type of resource still. Especially does this progress report happen on three separate occasions?

Reply:
Sure I can change it back then!  I was flip flopping about which way I preferred more.  It just happens so fast